### DISCUSSION TITLE
[Enhancement]: Improve Recently Added tab

### DISCUSSION THREAD

**Discussion by Naxterra** [ciderapp/Cider#702] [Feature Request]
### The feature you would like

In Library > Recently Added tab, it is not possible to skip to next or previous song. I have to select songs manually to play them.

#695 

Instead of playing songs one by one, enhance this tab because it looks like same as Library > Songs > Sort by Date with no distinction, or it can be removed from the app.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by cryptofyre** [comment]
Please use GitHub's Discussions for now on.
